punchr.proto

```diff
@@ -127,6 +127,11 @@ message HolePunchAttempt {
   // The error that occurred if the hole punch failed
   optional string error = 7;
 
+  // TODO mxinden: I am not quite sure I understand this field. Why does the
```

Comment:
@dennis-tra I might be missing something. Is this field still needed?

Reply:
This is just another code path that could in theory be reached from the Go implementation and I just wanted to be able to track all potential information from the hole punch. The most interesting research results come from things that should never happen but do anyways :D I would opt for keeping it and just rephrase the comment to make it unambiguously clear that this field should never have a value.